be/src/connector/hive_connector.cpp

```diff
@@ -256,6 +256,9 @@ Status HiveDataSource::_init_scanner(RuntimeState* state) {
     scanner_params.case_sensitive = _case_sensitive;
     scanner_params.profile = &_profile;
     scanner_params.open_limit = nullptr;
+    for (const auto& delete_file : scan_range.delete_files) {
```

Comment:
Another question is, how can you make sure these delete_files are associated the scan range file?

Reply:
A hive data source is associated to a scan range, how can you make sure `delete_files` are bind to this scan range? 

<img width="678" alt="image" src="https://user-images.githubusercontent.com/1081215/196107797-18e8968d-16a0-4be9-9e66-0eab8963ee4d.png">